ocaml/gencert/selfcert.ml

```diff
@@ -137,13 +137,17 @@ let host ~name ~dns_names ~ips pemfile =
   in
   R.failwith_error_msg res
 
+let random_string () = Random.int64 Int64.max_int |> Int64.to_string
```

Comment:
To help anybody who ever looks at these certs, maybe we should use a string that has a bit more information, like `"xapi timestamp %d"` and `%d` is a timestamp or `"xapi random %d"`.